tests/integration/product-type-import.spec.js

```diff
@@ -0,0 +1,174 @@
+import { SphereClient } from 'sphere-node-sdk'
+import Promise from 'bluebird'
+import test from 'tape'
+import ProductTypeImport from '../../src'
+import getSphereClientCredentials from '../../src/utils'
+
+const PROJECT_KEY = 'sphere-node-product-type-import'
```

Comment:
Done 0580a49ea7 and cannot be added to boilerplate because we don't have such test. Should be added to the contributing.md file.